sass/cd-header/_cd-dropdowns.scss

```diff
@@ -0,0 +1,92 @@
+/**
+ * Common Design Header Dropdowns generic styles.
+ */
+
+.cd-dropdown {
+  display: none;
+  .open > & {
+    display: block;
+  }
+}
+
+.open {
+  & > [data-toggle="dropdown"] {
+    & > .icon-arrow-down {
+      &:before {
+        content: "\e957"
+      }
+    }
+  }
+}
+
+/**
+ * Dropdown styles for nested drupal menus.
```

Comment:
These selectors are specific to Drupal at the moment, but I endeavor to reduce as much as possible. Or eliminate